payments-model/api/payments-model.api

```diff
@@ -187,6 +187,84 @@ public final class com/stripe/android/model/CardFunding : java/lang/Enum {
+	public static final field Expired Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public static final field Failed Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public static final field Started Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public static final field Unknown Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public static final field Verified Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public fun describeContents ()I
+	public final fun getValue ()Ljava/lang/String;
+	public static fun valueOf (Ljava/lang/String;)Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public static fun values ()[Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public fun writeToParcel (Landroid/os/Parcel;I)V
+}
+
+public final class com/stripe/android/model/ConsumerSession$VerificationSession$SessionState$Companion {
+	public final fun fromValue (Ljava/lang/String;)Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+}
+
+public final class com/stripe/android/model/ConsumerSession$VerificationSession$SessionState$Creator : android/os/Parcelable$Creator {
+	public fun <init> ()V
+	public final fun createFromParcel (Landroid/os/Parcel;)Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public synthetic fun createFromParcel (Landroid/os/Parcel;)Ljava/lang/Object;
+	public final fun newArray (I)[Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionState;
+	public synthetic fun newArray (I)[Ljava/lang/Object;
+}
+
+public final class com/stripe/android/model/ConsumerSession$VerificationSession$SessionType : java/lang/Enum, android/os/Parcelable {
+	public static final field CREATOR Landroid/os/Parcelable$Creator;
+	public static final field Companion Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType$Companion;
+	public static final field Email Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public static final field SignUp Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public static final field Sms Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public static final field Unknown Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public fun describeContents ()I
+	public final fun getValue ()Ljava/lang/String;
+	public static fun valueOf (Ljava/lang/String;)Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public static fun values ()[Lcom/stripe/android/model/ConsumerSession$VerificationSession$SessionType;
+	public fun writeToParcel (Landroid/os/Parcel;I)V
+}
+
+public final class com/stripe/android/model/ConsumerSession$VerificationSession$SessionType$Companion {
```

Comment:
Looks like we need a restrict to

Reply:
https://github.com/stripe/stripe-android/pull/6089/commits/d33f90f5904292edbee9fca9f0a2b6fbe47ec0f9